test/pixabay-spec.js

```diff
@@ -1,19 +1,20 @@
 'use strict';
 
-var chai = require('chai');
-var chaiAsPromised = require('chai-as-promised');
-var pixabay = require('../src/index');
-var ResultList = require('../src/result-list');
+import ResultList from '../src/result-list';
```

Comment:
I see you got this working. That is cool. But let's keep imports consistent. One or the other.